python-regression/tests/features/machine2/config.yml

```diff
@@ -6,7 +6,9 @@ defaults: &transaction_tests_config_files
   '--milestone-start',
   '0',
   '--testnet-no-coo-validation',
+  'true',
   '--testnet',
+  'true',
```

Comment:
Yes, I was under the impression that this would have been merged much sooner, and it would have fixed the issue then. Didn't seem like something that needed a new PR at the time, but since it didn't get merged as soon as I had hoped I agree I should have made this it's own.